doc/manual/syntax.md

```diff
@@ -200,6 +200,92 @@ def concat(str_array, log=false):
   return res"
 ```
 
+#### Symbolic strings
+
+Symbolic strings are another type of special strings in Nickel. For example, you
+could encounter the following code in a library for using Nickel with Nix:
```

Comment:
> the following code 

Is there supposed to be a code snippet after this paragraph?

Reply:
Ah, yes, I moved text around too much